jobserver/authorization/permissions.py

```diff
@@ -0,0 +1,7 @@
+cancel_job = "cancel-job"
+check_output = "check-output"
+invite_org_members = "invite-org-members"
+invite_project_members = "invite-project-members"
+publish_output = "publish-output"
+review_project = "review-project"
+run_job = "run-job"
```

Comment:
Is there any reason to to the _/- difference? Wonder if it's worth keeping them identical? i.e. _ in both symbol and string?

I do wonder if an Enum would be good here, for better repr in logging and tracebackes

Reply:
No, just a habit, completely agree in hindsight they should be the same.

I've never had great luck with Python's Enums, they whole `.value()` part of their API has always felt clunky and opaque to me.  I'm not sure what the repr would do extra here given this a readable string?

Reply:
Well, it could repr as `Permission(publish_output)` or similar, which is nicer. But happy to got with whatever, was just a thought.